src/cli.rs

```diff
@@ -1009,6 +1009,11 @@ fn safe_exit() -> ! {
     std::process::exit(0);
 }
 
+#[inline]
```

Comment:
There's no need to add this, "obvious" things are inlined automatically. Compilers really like inlining things.

You only need it if you want something to be inlined across crates, or if you think the compiler will guess wrong.

(It's harmless so there's no hurry to clean it up, and I did the same thing in `buffer.rs` before I knew this :slightly_smiling_face:)

Reply:
Oh I see. I will remember to remove the inline attribute here before making a release tonight.